lib/rules/object.js

```diff
@@ -93,6 +93,54 @@ class Keys extends Rule {
     }
 }
 
+class Pattern extends Rule {
+    compile(params) {
+        this._keyRegex = new RegExp(params.args[0]);
+        this._valueSchema = params.args[1];
+    }
+
+    validate(params, callback) {
+        const value = params.value;
+        if (value === undefined || value === null) {
+            return callback(this.error(params));
+        }
+
+        const todo = Object.keys(value).map((key) => {
```

Comment:
@connor4312 Jojen doesn't use the library "async" - it uses https://github.com/WatchBeam/jojen/blob/master/lib/util.js#L1